modcc/parser.cpp

```diff
@@ -1215,44 +1222,35 @@ expression_ptr Parser::parse_integer() {
 }
 
 expression_ptr Parser::parse_binop(expression_ptr&& lhs, Token op_left) {
```

Comment:
The `Token`type isn't trivially copyable because `std::string`. It is being passed by value here: would passing by const reference also work From a cursory glance at the code, it looks like it would.